magda-web-client/src/Components/Dataset/DatasetPage.tsx

```diff
@@ -169,6 +169,14 @@ const DatasetPage: FunctionComponent<PropsType> = props => {
             </div>
         </div>
     );
+
+    useEffect(() => {
+        if (props.history.location.search === "?print=true") {
```

Comment:
Might want to use `query-string` for this (see the Account component for an example).

The problem with doing it like this is that if anyone every includes another query parameter and puts it first (e.g. `?foo=bar&print=true` this won't work anymore.